CRD-DTR/PositiveAirwayPressureDevices/R4/resources/Questionnaire-R4-PositiveAirwayPressureDevicesFaceToFace.json

```diff
@@ -217,13 +223,27 @@
         },
         {
           "linkId": "6.2",
-          "text": "Was E0601 (CPAP) tried not effective on therapeutic trial conducted above ",
-          "type": "boolean"
+          "text": "Was E0601 (CPAP) not effective on therapeutic trial conducted above ",
+          "type": "boolean",
+          "required": true,
+          "enableWhen": [
+            {
+              "question": "6",
+              "operator": "=",
+              "answerBoolean": "a home setting"
```

Comment:
this is giving me an error when loading the cds library.  I believe it should be an `answerString`.